fvm/fvm_test.go

```diff
@@ -85,6 +95,60 @@ func (vmt vmTest) run(
 	}
 }
 
+// bootstrapWith executes the bootstrap procedure and the custom bootstrap function
```

Comment:
maybe move this to fuzz test if you're not going to reuse it elsewhere

Reply:
Using this, the fvm tests, especially those that repeat the same setup, could be much faster. I think I will be using this in the future.